components/raftstore/src/store/fsm/apply.rs

```diff
@@ -1201,7 +1202,18 @@ where
         // Include region for epoch not match after merge may cause key not in range.
         let include_region =
             req.get_header().get_region_epoch().get_version() >= self.last_merge_version;
-        check_region_epoch(req, &self.region, include_region)?;
+        check_region_epoch(req, &self.region, include_region).map_err(|mut e| {
```

Comment:
After https://github.com/tikv/tikv/pull/8441, it's very unlikely the epoch-not-match happens here. And I will totally eliminate the epoch-not-match error during applying. So this change is needless.